setup.py

```diff
@@ -273,7 +273,7 @@ def get_data_files():
     },
     'entry_points': {
         'console_scripts': [
-            'voila = voila.app:main'
+            'jupyter-voila = voila.app:main'
```

Comment:
Do we want this in addition to `voila` for backward compatibility?

Reply:
We will probably want to have the standalone mode supported in `jupyter_server` before we make  `voila`  be the same thing as `jupyter-voila`.

Reply:
:+1: I'll probably work on the standalone mode later this week.

Reply:
added "standalone" mode in jupyter/jupyter_server#70